frontend/src/components/AnnouncementCard.tsx

```diff
@@ -1,16 +1,42 @@
+"use client";
+import deleteAnnouncement from "@/libs/deleteAnnouncement";
 import { AnnouncementItem, AnnouncementJson } from "interface";
+import { useSession } from "next-auth/react";
 
-export default async function AnnouncementCard({
+export default function AnnouncementCard({
   title,
   campground,
   content,
   endDate,
+  announcementId,
 }: {
   title: string;
   campground: string;
   content: string;
   endDate: Date;
+  announcementId: string;
 }) {
+  const { data: session } = useSession();
+
+  const onSubmitDelete = () => {
+    try {
+      const deleteOne = async () => {
+        await deleteAnnouncement(announcementId, session.user.token);
+      };
+      deleteOne();
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const confirmDelete = (e) => {
+    e.stopPropagation();
+    if (window.confirm("Are sure want to delete?")) {
```

Comment:
ตรงนี้แน่ว่าประโยคแอบแปกๆนิดนึงงงง ถ้าเปลี่ยนเป็น 'Are you sure you want to delete this item?' หรือ 'Do you confirm your deletion?' หรืออะไรคล้ายๆแบบนี้จะดีกว่าม้ายยย ที่เหลือโอเคแล้วคับบบ **Good job mak mak my cherri**

Reply:
โอเคค้าบ พอดีเค้าไปก้อปมาจากในเน็ตแล้วลืมอ่าน ;-;